src/components/entity/state-status.html

```diff
@@ -0,0 +1,33 @@
+<link rel="import" href="../../../bower_components/polymer/polymer.html">
+
+<dom-module id="state-status">
+  <template>
+    <style>
```

Comment:
This component should not include any CSS. It should just render the text. Let the styling be done in whatever incorporates this element.